controller/nc_news.controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require("../model/nc_news.model");
+
+exports.getTopics = (req, res, next) => {
+  fetchTopics()
+    .then((result) => {
+      res.status(200).send({ result });
```

Comment:
result is a vague key to store topics on within the response body, what key name could be used instead to refer to what data we are retrieving?